### PR TITLE
Improve website design 

### PR DESCRIPTION
**Pull request by KSNINJA**
- [x] Included a Preview link and screenshot showing after and before the changes.
- [x] Included a description of the change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- [Preview Link](https://ksninja.github.io/gci18.fossasia.org/)

- Fixes - #593 

## Description / Changes
Changed section heading theme ( color , font-weight , font-size  ).
Added image for social media section and changed image of welcome section .
Added scroll reveal feature .

### Before:
image
image
image

### After:
image
image
image

- - - - - - - - - - - -

### CODE REVIEW

**Comment by liveHarshit** [comment]
Same in students section.

**Comment by KSNINJA** [comment]
@liveHarshit I have made changes to font-size . I think that colour theme are at its best . If you have a better idea . Please tell me , I will make the changes .
For the issue , this is just an update . It is not related to any issue .

**Comment by liveHarshit** [comment]
Font for the title should be in bold and no style for subtitle. Also, reduce the size of the subtitle.
Open a new issue for the same, and link PR to the issue.

File: index.html [hunk 1]
```diff
@@ -403,24 +405,25 @@ <h4><b>{{video.name}}</b> by <b>{{video.user}}</b></h4>
 </section>
 <!-- Videos section -->
 <section class="lazy-section">
-  <div class="container" id="mentors">
+  <div class="container-fluid" id="mentors">
     <div class="row mb64 mb-xs-24">
       <div class="col-sm-12 col-md-10 col-md-offset-1 text-center">
-        <h3>Our Mentors</h3>
+        <h2 class="section-heading">Our Mentors</h2>
+        <h2 class="section-sub-heading">Anytime , Anywhere. Just Ask</h2>
```
Comment (jakubpawlina) [inline index.html]: Remove the space before comma

**Comment by fragm3** [comment]
Make the style for `Map, Student, Our mentors` same as done for `Contributors`, you can make the subheading grey if it looks nice.

**Comment by KSNINJA** [comment]
I have made the changes . Please review now .

**Comment by KSNINJA** [comment]
I made the changes.  Please review.

**Pull request merged**